src/components/tasks/card/index.tsx

```diff
@@ -488,13 +492,7 @@ const Card: FC<Props> = ({
             {isLoading && <Loader />}
 
             <div className={classNames.cardItems}>
-                <Link
-                    href={{
-                        pathname: '/tasks/[id]',
-                    }}
-                    as={`/tasks/${cardDetails.id}`}
-                    style={{ textDecoration: 'none' }}
-                >
+                <CardTitleWrapper condition={isNewCardEnabled}>
```

Comment:
I dont see the other 2 props `to` and `taskId` passed here ?

Reply:
We don't need them here, as the CardTitle in this case will not be a link. So, we don't need to pass the `to` and `taskId`